src/unix/linux-core.c

```diff
@@ -193,9 +193,11 @@ void uv__io_poll(uv_loop_t* loop, int timeout) {
   int count;
   int nfds;
   int fd;
-  int op;
+  int op = 0;
   int i;
```

Comment:
Yes, i got uninitialized value warning in valgrind without those. I already had these changes in my fork.

What about the rest of the PR?